datajoint/blob.py

```diff
@@ -14,33 +14,33 @@
 from .settings import config
 
 
-mxClassID = dict(
+scalar_id = dict(
     (
         # see http://www.mathworks.com/help/techdoc/apiref/mxclassid.html
-        ("mxUNKNOWN_CLASS", None),
-        ("mxCELL_CLASS", None),
-        ("mxSTRUCT_CLASS", None),
-        ("mxLOGICAL_CLASS", np.dtype("bool")),
-        ("mxCHAR_CLASS", np.dtype("c")),
-        ("mxVOID_CLASS", np.dtype("O")),
-        ("mxDOUBLE_CLASS", np.dtype("float64")),
-        ("mxSINGLE_CLASS", np.dtype("float32")),
-        ("mxINT8_CLASS", np.dtype("int8")),
-        ("mxUINT8_CLASS", np.dtype("uint8")),
-        ("mxINT16_CLASS", np.dtype("int16")),
-        ("mxUINT16_CLASS", np.dtype("uint16")),
-        ("mxINT32_CLASS", np.dtype("int32")),
-        ("mxUINT32_CLASS", np.dtype("uint32")),
-        ("mxINT64_CLASS", np.dtype("int64")),
-        ("mxUINT64_CLASS", np.dtype("uint64")),
-        ("mxFUNCTION_CLASS", None),
-        ("NON_mx_DATETIME64", np.dtype("<M8[us]")),
+        ("UNKNOWN", None),
+        ("CELL", None),
+        ("STRUCT", None),
+        ("LOGICAL", np.dtype("bool")),
+        ("CHAR", np.dtype("c")),
+        ("VOID", np.dtype("O")),
+        ("DOUBLE", np.dtype("float64")),
+        ("SINGLE", np.dtype("float32")),
+        ("INT8", np.dtype("int8")),
+        ("UINT8", np.dtype("uint8")),
+        ("INT16", np.dtype("int16")),
+        ("UINT16", np.dtype("uint16")),
+        ("INT32", np.dtype("int32")),
+        ("UINT32", np.dtype("uint32")),
+        ("INT64", np.dtype("int64")),
+        ("UINT64", np.dtype("uint64")),
+        ("FUNCTION", None),
+        ("DATETIME64", np.dtype("<M8[us]")),
```

Comment:
since matlab uses the specific numeric codes from this sequence (0-16), perhaps we could specify them here explicitly and skip some space to allow matlab to add their additional types before we introduce our type such as `datetime64`.